server/@types/forms/index.d.ts

```diff
@@ -124,6 +124,11 @@ declare module 'inductionForms' {
     educationLevel: EducationLevelValue
   }
 
+  export interface QualificationLevelForm {
+    educationLevel: EducationLevelValue
```

Comment:
Why does the form need the education level? We currently have that modelled in it's own form `HighestLevelOfEducationForm` ... 🤔

Reply:
The logic in the template uses it. For example:

```
{% if ['FURTHER_EDUCATION_COLLEGE', 'UNDERGRADUATE_DEGREE_AT_UNIVERSITY', 'POSTGRADUATE_DEGREE_AT_UNIVERSITY'].includes(form.educationLevel) or form.educationLevel === '' %}
  {% set title = "What level of further education college qualification does " + prisonerSummary.firstName + " " + prisonerSummary.lastName + " want to add?" %}
```

Reply:
Ah yes, that makes sense 👍